libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1083,7 +1083,8 @@ bool ExpressionCompiler::visit(FunctionCall const& _functionCall)
 					targetTypes.emplace_back(argument->annotation().type);
 				else if (
 					auto const* literalType = dynamic_cast<StringLiteralType const*>(argument->annotation().type);
-					literalType && literalType->value().size() <= 32
+					literalType &&
+					0 < literalType->value().size() && literalType->value().size() <= 32
```

Comment:
I think `>= 1` is more readable for me at least:
```suggestion
					literalType->value().size() >= 1 && literalType->value().size() <= 32
```

Or
```suggestion
					!literalType->value().empty() && literalType->value().size() <= 32
```

Reply:
True. Agreed on `>= 1` vs `>0`.

As to the order, I changed it but I find the original one more readable when I'm checking that a variable is in a certain range. It's closer to `0 <= x < N` which is more natural to me.